plagiarism/process.go

```diff
@@ -0,0 +1,136 @@
+package plagiarism
+
+import (
+    "io"
+    "io/ioutil"
+    "encoding/json"
+    "log"
+    "github.com/SpaceWhite/gomoss"
+    "fmt"
+    "os"
+)
+
+// GUIDsStruct use for decoding JSON
+type GUIDsStruct struct {
+    GUIDs []string
+}
+
+// Analysis struct contains output dir and reference dir
+type Analysis struct {
+    GIUDs []string
+    SubmissionDir string
+    Mossid uint32
+    Logger *log.Logger
+    ResultDir string
+}
+
+// DecodeRequest decode request from request body
+func DecodeRequest(body io.ReadCloser) ([]string, error) {
+    decoder := json.NewDecoder(body)
+    var guidsStruct GUIDsStruct
+    err := decoder.Decode(&guidsStruct)
+
+    if err != nil {
+        return []string{}, err
+    }
+    return guidsStruct.GUIDs, nil
+}
+
+// GUIDPath use to get submission from OmegaUp
```

Comment:
nit: all the documentation should be in present tense (e.g. "GUIDPath returns the path of a submission in the filesystem, given the submission GUID")